crates/hc_deepkey_sdk/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+pub use hc_deepkey_types::*;
+
+pub use hdk;
```

Comment:
Is this still useful?

Reply:
I am using it here: https://github.com/holochain/holochain/blob/b27d41cc515524f7819718b554c8f0f2339b6aa8/crates/holochain_types/src/lib.rs#L62

Which is a strange thing to do, because it's just deserializing to the same type version, but this was the quickest way to integrate this stuff that I could find. I think it will become more relevant once implementing this TODO: https://github.com/holochain/holochain/blob/ccef30f4daf49d082694cc306ab666b8763da086/crates/hc_deepkey_sdk/Cargo.toml#L19